src/components/gridcapa-main.js

```diff
@@ -95,6 +115,15 @@ const GridCapaMain = () => {
                             data-test="global-view"
                         />
                     </Tabs>
+                    <div class="center">
+                        <FormattedMessage id="minioDiskUsage" />
+                        {Math.round(usedDiskSpacePercentage)}%
```

Comment:
Why do the rounding here and not directly during the calculation of 'usedDiskSpacePercentage'